sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -1,4 +1,45 @@
 # Release History
 
+## 1.0.0b2 (2022-01-20)
+
+### Features Added
+    
+  - Added DeviceManagementOperations
+  - Added DeviceUpdateOperations
+  - Added operation `send_request` to send customized http request conveniently
+
+### Breaking Changes
+
+  - Removed all models
+  - Removed DeploymentsOperations
+  - Removed UpdateOperations
+  - Dropped support for Python2.7(https://github.com/Azure/azure-sdk-for-python#disclaimer)
+  - Parameter `account_endpoint` of DeviceUpdateClient is renamed to `endpoint`
+  - Renamed UpdatesOperations.get_update to DeviceUpdateOperations.get_update
```

Comment:
sorry if I wasn't clear, I don't think we need to list every operation as being changed between these two operations (unless they actually have). I think something along the lines of `client.updates` renamed to `client.device_update` would be clearer and easier to read